src/ext/line-numbers.lisp

```diff
@@ -15,7 +17,15 @@
 (defvar *previous-relative-line* nil)
 
 (defvar *initialized* nil)
-(defvar *line-number-format* nil)
+
+(define-editor-variable line-number-format "~6D "
+  "Set to desired format, for example, \"~2D \" for a
+two-character line-number column.")
+
+(define-editor-variable relative-line-numbers-current-line-value 
+  '(line-number-at-point (current-point))
+  "Set to desired current-line value when relative line numbers
```

Comment:
I think it would be better to make this a function rather than eval.

Reply:
Before that, it seems like the API that specifies this behavior from the outside is not a good idea.